### PR TITLE
Create RVM repo TF file

### PR DESCRIPTION
**Pull request by martinisoft**
## Description

Sets up myself as an RVM repo maintainer. Will still need help fixing the name of the repo after transfer.

### Issues Resolved

- None

### Check List
- [ ] All tests pass. See https://github.com/sous-chefs/apache2/blob/master/TESTING.md
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable

### CODE REVIEW

File: rvm.tf [hunk 1]
```diff
@@ -0,0 +1,17 @@
+module "rvm" {
+  source        = "modules/repository"
+  name          = "rvm"
+  cookbook_team = "${github_team.nginx.id}"
```
Comment (JohnRoesler) [inline rvm.tf]: probably want this to be the rvm team

File: rvm.tf [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+}
+
+resource "github_team" "rvm" {
+  name        = "rvm"
+  description = "rvm Cookbook Maintainers"
+  privacy     = "closed"
+}
+
+resource "github_team_membership" "rvm-maintainer-1" {
+  team_id  = "${github_team.nginx.id}"
```
Comment (JohnRoesler) [inline rvm.tf]: and rvm team id here

File: rvm.tf (hunk 1 above)
Comment (martinisoft) [reply rvm.tf]: Good catch, I suck at renaming things :)

File: rvm.tf (hunk 2 above)
Comment (martinisoft) [reply rvm.tf]: also good catch

**Comment by damacus** [comment]
@martinisoft shouldn't this have been chef-rvm?

**Comment by martinisoft** [comment]
@damacus I need you or an org admin to rename the repo since I lost ownership when I transferred it (didn't set it up right when I did)

**Pull request merged**